fintrack/users/models.py

```diff
@@ -4,3 +4,4 @@
 
 class User(AbstractUser):
     address = models.CharField(max_length=250)
+    email = models.EmailField(unique=True, blank=False)
```

Comment:
weren't we going to use username instead of email? or are we thinking about adding forgot password feature?

Reply:
yes, purpose of making email unique was adding forgot password feature as also mentioned in SRS. Still username is also unique for every user.